firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nBaseSnapshotTests.swift

```diff
@@ -131,4 +131,11 @@ extension XCUIElement {
     func tapOnApp() {
         coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.5)).tap()
     }
+
+    func waitAndTap(timeout: TimeInterval? = TIMEOUT) {
+        L10nBaseSnapshotTests().mozWaitForElementToExist(self, timeout: timeout)
+        if self.exists {
```

Comment:
Can we add some code to improve the readability and check if the condition isn't met, terminate early the method? I think we can improve this with the code

`
guard self.exists else {
  XCTFail("Element not found whithin \(timeout ?? 0) seconds")
            return 
}
// Tap the element if it exists
self.tap()
`

or with an if else, like this:
`if self.exists {
        self.tap()
    } else {
        XCTFail("Element not found within \(timeout ?? 0) seconds")
    }`

What do you think?

Reply:
Updated `mozWaitForElementToExist` function to use `guard`. Thanks for the tip 🙏🏽 
Fixed here: https://github.com/mozilla-mobile/firefox-ios/pull/22528/commits/a50c8636d5aaee960a6340cd482fff64644695fc